src/pybind/mgr/rook/module.py

```diff
@@ -374,7 +374,7 @@ def _list_daemons(self, service_name=None, daemon_type=None, daemon_id=None, hos
         for p in pods:
             sd = orchestrator.DaemonDescription()
             sd.hostname = p['hostname']
-            sd.container_id = p['name']
+            sd.container_id = p['container_id'][0:12]
```

Comment:
What are the `container_id` and `container_image_id` used for?

Reply:
They are just displayed in output of 'orch ps' and 'orch ls' commands.

```
# ceph orch ps
NAME   HOST      STATUS         REFRESHED  AGE  VERSION    IMAGE NAME                           IMAGE ID      CONTAINER ID          
mgr.a  minikube  Running (89s)  0s ago     89s  <unknown>  192.168.0.138:5000/ceph/ceph:latest  cc82ea0cb1ef  docker://c5555d21c3c  
mon.a  minikube  Running (79s)  0s ago     79s  <unknown>  192.168.0.138:5000/ceph/ceph:latest  cc82ea0cb1ef  docker://7f689c1f1a4  
osd.0  minikube  Running (72m)  0s ago     72m  <unknown>  192.168.0.138:5000/ceph/ceph:latest  106dcc228585  docker://ce7fce2fb92  

# ceph orch ls
NAME  RUNNING  REFRESHED  AGE  PLACEMENT  IMAGE NAME                           IMAGE ID      
mgr       1/1  0s ago     98s  count:1    192.168.0.138:5000/ceph/ceph:latest  cc82ea0cb1ef  
mon       1/1  0s ago     88s  count:1    192.168.0.138:5000/ceph/ceph:latest  cc82ea0cb1ef  
```

Reply:
My question is: it it worth it to display the container id? The image id is interesting to know which ceph container is being used. but why bothering with the container id?

Reply:
I don't think so. I kept container id just to maintain consistent interface. Do you plan to remove it completely from the orchestrator module ?